### ISSUE TITLE
Crash when generating world

### ISSUE THREAD

**Issue by Shazuli** [jbredwards/Fluidlogged-API#120]
**Describe the bug**
The game crashed while generating a new world.

**Logs**
<details>
  <summary>crash-2022-09-01_19.04.22-server.txt</summary>

```
---- Minecraft Crash Report ----
// Lolis deobfuscated this stacktrace using MCP's stable-39 mappings.
// Ooh. Shiny.

Time: 2022-09-01 19:04:22 CEST
Description: Exception while ticking a block

java.lang.IncompatibleClassChangeError
    at git.jbredwards.fluidlogged_api.api.util.FluidloggedUtils.getFluidFromState(FluidloggedUtils.java:218)
    at git.jbredwards.fluidlogged_api.mod.asm.plugins.ASMHooks.setBlockState(ASMHooks.java:1631)
    at net.minecraft.world.World.setBlockState(World.java:343)
    at net.minecraft.block.BlockDynamicLiquid.placeStaticBlock(BlockDynamicLiquid.java:24)
    at git.jbredwards.fluidlogged_api.mod.asm.plugins.ASMHooks.updateLiquidTick(ASMHooks.java:845)
    at net.minecraft.block.BlockDynamicLiquid.updateTick(BlockDynamicLiquid.java)
    at net.minecraft.world.WorldServer.tickUpdates(WorldServer.java:737)
    at net.minecraft.world.WorldServer.tick(WorldServer.java:223)
    at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:756)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668)
    at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer.java:185)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
    at java.lang.Thread.run(Thread.java:748)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Block being ticked --
  Block: minecraft:flowing_lava[level=2]
  Block location: World: (-353,22,304), Chunk: (at 15,1,0 in -23,19; contains blocks -368,0,304 to -353,255,319), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)

-- Affected level --
  Level name: New World
  All players: 1 total; [GCEntityPlayerMP['simon_kungen'/9111, l='New World', x=-147.73, y=66.13, z=281.69]]
  Chunk stats: ServerChunkCache: 849 Drop: 0
  Level seed: 8803563413085070181
  Level generator: ID 00 - default, ver 1. Features enabled: true
  Level generator options: {"diamondSize":8,"lapisCenterHeight":16,"lavaLakeChance":80,"redstoneMaxHeight":16,"diamondMaxHeight":16,"coalMaxHeight":128,"coalMinHeight":0,"seaLevel":63,"graniteCount":10,"riverSize":4,"gravelMinHeight":0,"graniteSize":33,"redstoneSize":8,"fixedBiome":-1,"waterLakeChance":4,"dioriteMinHeight":0,"andesiteCount":10,"graniteMaxHeight":80,"redstoneMinHeight":0,"lapisSize":7,"dirtCount":10,"ironMinHeight":0,"dioriteMaxHeight":80,"coalCount":20,"gravelCount":8,"coalSize":17,"gravelSize":33,"ironMaxHeight":64,"lapisSpread":16,"dioriteCount":10,"andesiteSize":33,"biomeSize":4,"graniteMinHeight":0,"goldCount":2,"diamondCount":1,"dirtSize":33,"gravelMaxHeight":256,"goldSize":9,"ironCount":20,"diamondMinHeight":0,"ironSize":9,"redstoneCount":8,"andesiteMinHeight":0,"dioriteSize":33,"goldMaxHeight":32,"dungeonChance":8,"andesiteMaxHeight":80,"dirtMinHeight":0,"lapisCount":1,"goldMinHeight":0,"dirtMaxHeight":256,"lowerLimitScale":512.0,"biomeDepthOffset":0.0,"upperLimitScale":512.0,"heightScale":684.412,"biomeScaleWeight":1.0,"biomeDepthWeight":1.0,"mainNoiseScaleZ":80.0,"mainNoiseScaleY":160.0,"depthNoiseScaleX":200.0,"mainNoiseScaleX":80.0,"baseSize":8.5,"depthNoiseScaleZ":200.0,"stretchY":12.0,"depthNoiseScaleExponent":0.5,"biomeScaleOffset":0.0,"coordinateScale":684.412,"useStrongholds":true,"useMineShafts":true,"useTemples":true,"useLavaOceans":false,"useCaves":true,"useWaterLakes":true,"useLavaLakes":true,"useMonuments":true,"useRavines":true,"useDungeons":true,"useVillages":true}
  Level spawn location: World: (-132,64,256), Chunk: (at 12,4,0 in -9,16; contains blocks -144,0,256 to -129,255,271), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
  Level time: 57 game time, 57 day time
  Level dimension: 0
  Level storage version: 0x04ABD - Anvil
  Level weather: Rain time: 52798 (now: false), thunder time: 32928 (now: false)
  Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: true

-- System Details --
  Minecraft Version: 1.12.2
  Operating System: Linux (amd64) version 5.15.60-1-MANJARO
  Java Version: 1.8.0_311, Oracle Corporation
  Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
  Memory: 1422741192 bytes (1356 MB) / 3742367744 bytes (3569 MB) up to 3817865216 bytes (3641 MB)
  JVM Flags: 2 total; -Xms1024m -Xmx4096m
  IntCache: cache: 12, tcache: 94, allocated: 0, tallocated: 0
  FML: MCP 9.42 LoliASM 5.6 Powered by Forge 14.23.5.2860 131 mods loaded, 131 mods active
       States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
       
       | State  | ID                           | Version                            | Source                                                     | Signature                                |
       |:------ |:---------------------------- |:---------------------------------- |:---------------------------------------------------------- |:---------------------------------------- |
       | LCHIJA | minecraft                    | 1.12.2                             | minecraft.jar                                              | None                                     |
       | LCHIJA | mcp                          | 9.42                               | minecraft.jar                                              | None                                     |
       | LCHIJA | FML                          | 8.0.99.99                          | forge-1.12.2-14.23.5.2860-universal.jar                    | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJA | forge                        | 14.23.5.2860                       | forge-1.12.2-14.23.5.2860-universal.jar                    | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJA | creativecoredummy            | 1.0.0                              | minecraft.jar                                              | None                                     |
       | LCHIJA | itemphysic                   | 1.4.0                              | minecraft.jar                                              | None                                     |
       | LCHIJA | micdoodlecore                |                                    | minecraft.jar                                              | None                                     |
       | LCHIJA | persistency                  | 1.2.0                              | minecraft.jar                                              | None                                     |
       | LCHIJA | smoothfontcore               | mc1.12.2-2.1.2                     | minecraft.jar                                              | None                                     |
       | LCHIJA | mixinbooter                  | 5.0                                | minecraft.jar                                              | None                                     |
       | LCHIJA | opencomputers|core           | 1.7.6+fe5ba4e                      | minecraft.jar                                              | None                                     |
       | LCHIJA | randompatches                | 1.12.2-1.22.1.10                   | randompatches-1.12.2-1.22.1.10.jar                         | None                                     |
       | LCHIJA | smoothfont                   | mc1.12.2-2.1.2                     | SmoothFont-mc1.12.2-2.1.2.jar                              | None                                     |
       | LCHIJA | witchery                     | 1.12.2-0.5.2.4-forge               | WitcheryResurrected-1.12.2-0.5.2.4-forge-all.jar           | None                                     |
       | LCHIJA | aiimprovements               | 0.0.1.1                            | AIImprovements-1.12.1-0.0.1b1.jar                          | None                                     |
       | LCHIJA | akashictome                  | 1.2-12                             | AkashicTome-1.2-12.jar                                     | None                                     |
       | LCHIJA | applecore                    | 3.4.0                              | AppleCore-mc1.12.2-3.4.0.jar                               | None                                     |
       | LCHIJA | ctm                          | MC1.12.2-1.0.2.31                  | CTM-MC1.12.2-1.0.2.31.jar                                  | None                                     |
       | LCHIJA | appliedenergistics2          | rv6-stable-7-extended_life-v0.52.7 | appliedenergistics2-rv6-stable-7-extended_life-v0.52.7.jar | None                                     |
       | LCHIJA | aquaacrobatics               | 1.12.1                             | AquaAcrobatics-1.12.1.jar                                  | None                                     |
       | LCHIJA | armorpointspp                | v2.0.0-rc1                         | armorpointspp-1.12.2-2.0.1.jar                             | None                                     |
       | LCHIJA | galacticraftcore             | 4.0.2.280                          | GalacticraftCore-1.12.2-4.0.2.280.jar                      | None                                     |
       | LCHIJA | galacticraftplanets          | 4.0.2.280                          | Galacticraft-Planets-1.12.2-4.0.2.280.jar                  | None                                     |
       | LCHIJA | asmodeuscore                 | 0.0.30                             | AsmodeusCore-1.12.2-0.0.30.jar                             | None                                     |
       | LCHIJA | crafttweaker                 | 4.1.20                             | CraftTweaker2-1.12-4.1.20.673.jar                          | None                                     |
       | LCHIJA | mtlib                        | 3.0.7                              | MTLib-3.0.7.jar                                            | None                                     |
       | LCHIJA | modtweaker                   | 4.0.19                             | modtweaker-4.0.20.4.jar                                    | None                                     |
       | LCHIJA | jei                          | 4.24.0                             | HadEnoughItems_1.12.2-4.24.0.jar                           | None                                     |
       | LCHIJA | quark                        | r1.6-179                           | Quark-r1.6-179.jar                                         | None                                     |
       | LCHIJA | autoreglib                   | 1.3-32                             | AutoRegLib-1.3-32.jar                                      | None                                     |
       | LCHIJA | baubles                      | 1.5.2                              | Baubles-1.12-1.5.2.jar                                     | None                                     |
       | LCHIJA | bettermineshafts             | 1.12.2-2.2.1                       | BetterMineshaftsForge-1.12.2-2.2.1.jar                     | None                                     |
       | LCHIJA | betternether                 | 0.1.9.4                            | BetterNetherRotN-1.12.2-0.1.9.4.jar                        | None                                     |
       | LCHIJA | bloodmoon                    | 1.5.3                              | Bloodmoon-MC1.12.2-1.5.3.jar                               | d72e0dd57935b3e9476212aea0c0df352dd76291 |
       | LCHIJA | blur                         | 1.0.4-14                           | Blur-1.0.4-14.jar                                          | None                                     |
       | LCHIJA | bookshelf                    | 2.3.577                            | Bookshelf-1.12.2-2.3.577.jar                               | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJA | thaumcraft                   | 6.1.BETA26                         | Thaumcraft-1.12.2-6.1.BETA26.jar                           | None                                     |
       | LCHIJA | botania                      | r1.10-364                          | Botania+r1.10-364.4.jar                                    | None                                     |
       | LCHIJA | carrots                      | 1.0.0b1                            | carrotslib-mc1.12.2-1.0.0b1.jar                            | None                                     |
       | LCHIJA | chameleon                    | 1.12-4.1.3                         | Chameleon-1.12-4.1.3.jar                                   | None                                     |
       | LCHIJA | charm                        | 1.4                                | Charm-1.12.2-1.4.1.jar                                     | None                                     |
       | LCHIJA | chunkanimator                | 1.2                                | ChunkAnimator-MC1.12-1.2.jar                               | None                                     |
       | LCHIJA | codechickenlib               | 3.2.3.357                          | CodeChickenLib-1.12.2-3.2.3.357-universal.jar              | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | LCHIJA | asielib                      | 1.0.0                              | Computronics-1.12.2-1.6.6.jar                              | None                                     |
       | LCHIJA | endercore                    | 1.12.2-0.5.76                      | EnderCore-1.12.2-0.5.76.jar                                | None                                     |
       | LCHIJA | enderio                      | 5.3.70                             | EnderIO-base-1.12.2-5.3.70.jar                             | None                                     |
       | LCHIJA | forestry                     | 5.8.2.387                          | forestry_1.12.2-5.8.2.387.jar                              | None                                     |
       | LCHIJA | gregtech                     | 2.4.1-beta                         | gregtech-1.12.2-2.4.1-beta.jar                             | None                                     |
       | LCHIJA | opencomputers                | 1.7.6+fe5ba4e                      | OpenComputers-MC1.12.2-1.7.6+fe5ba4e.jar                   | None                                     |
       | LCHIJA | tis3d                        | 1.5.1.37                           | TIS-3D-MC1.12.2-1.5.1.37.jar                               | None                                     |
       | LCHIJA | computronics                 | 1.6.6                              | Computronics-1.12.2-1.6.6.jar                              | None                                     |
       | LCHIJA | consecration                 | 1.0.6                              | consecration-1.12.2-1.0.6.jar                              | 2484ef4d131fdc0dca0647aa21b7b944ddb935a1 |
       | LCHIJA | ctgui                        | 1.0.0                              | CraftTweaker2-1.12-4.1.20.673.jar                          | None                                     |
       | LCHIJA | crafttweakerjei              | 2.0.3                              | CraftTweaker2-1.12-4.1.20.673.jar                          | None                                     |
       | LCHIJA | creativecore                 | 1.10.0                             | CreativeCore_v1.10.61_mc1.12.2.jar                         | None                                     |
       | LCHIJA | cyberware                    | 0.2.11.29                          | cyberware-1.12.2-0.2.11.29.jar                             | None                                     |
       | LCHIJA | damagetilt                   | 0.1.0                              | DamageTilt-1.12.2-0.1.0.jar                                | None                                     |
       | LCHIJA | eleccore                     | 1.9.452                            | ElecCore-1.12.2-1.9.452.jar                                | None                                     |
       | LCHIJA | mcjtylib_ng                  | 3.5.0                              | mcjtylib-1.12-3.5.0.jar                                    | None                                     |
       | LCHIJA | deepresonance                | 1.8.0                              | deepresonance-1.12-1.8.0.jar                               | None                                     |
       | LCHIJA | devtech                      | 2.0.4                              | devtech-2.0.4.jar                                          | None                                     |
       | LCHIJA | diethopper                   | 1.1                                | diethopper-1.1.jar                                         | None                                     |
       | LCHIJA | enderiobase                  | 5.3.70                             | EnderIO-base-1.12.2-5.3.70.jar                             | None                                     |
       | LCHIJA | enderioconduits              | 5.3.70                             | EnderIO-conduits-1.12.2-5.3.70.jar                         | None                                     |
       | LCHIJA | enderiopowertools            | 5.3.70                             | EnderIO-conduits-1.12.2-5.3.70.jar                         | None                                     |
       | LCHIJA | enderioconduitsopencomputers | 5.3.70                             | EnderIO-conduits-opencomputers-1.12.2-5.3.70.jar           | None                                     |
       | LCHIJA | enderiointegrationforestry   | 5.3.70                             | EnderIO-integration-forestry-1.12.2-5.3.70.jar             | None                                     |
       | LCHIJA | enderiomachines              | 5.3.70                             | EnderIO-machines-1.12.2-5.3.70.jar                         | None                                     |
       | LCHIJA | entityculling                | 5.2.2                              | EntityCulling-1.12.2-5.2.2.jar                             | None                                     |
       | LCHIJA | ae2fc                        | 2.3.3-r                            | Fluid+Craft+for+AE2-2.3.3-r.jar                            | None                                     |
       | LCHIJA | fluidlogged_api              | 1.8.1e                             | Fluidlogged-API-v1.8.1e-mc1.12.2.jar                       | None                                     |
       | LCHIJA | forgemultipartcbe            | 2.6.2.83                           | ForgeMultipart-1.12.2-2.6.2.83-universal.jar               | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | LCHIJA | microblockcbe                | 2.6.2.83                           | ForgeMultipart-1.12.2-2.6.2.83-universal.jar               | None                                     |
       | LCHIJA | minecraftmultipartcbe        | 2.6.2.83                           | ForgeMultipart-1.12.2-2.6.2.83-universal.jar               | None                                     |
       | LCHIJA | cfm                          | 6.3.1                              | furniture-6.3.1-1.12.2.jar                                 | None                                     |
       | LCHIJA | galacticrafttweaker          | 1.12.2-1.0.3                       | GalacticraftTweaker-1.12.2-1.0.3.jar                       | b02331787272ec3515ebe63ecdeea0d746653468 |
       | LCHIJA | galaxyspace                  | 2.0.18                             | GalaxySpace-1.12.2-2.0.18.jar                              | None                                     |
       | LCHIJA | gcym                         | 1.1.4                              | GregicalityMultiblocks-1.12.2-1.1.4.jar                    | None                                     |
       | LCHIJA | gtce2oc                      | 1.8.0                              | gtce2oc-1.12.2-1.8.0.jar                                   | None                                     |
       | LCHIJA | hardcoredarkness             | 2.0                                | HardcoreDarkness-MC1.12.2-2.0.jar                          | d72e0dd57935b3e9476212aea0c0df352dd76291 |
       | LCHIJA | ichunutil                    | 7.2.2                              | iChunUtil-1.12.2-7.2.2.jar                                 | 4db5c2bd1b556f252a5b8b54b256d381b2a0a6b8 |
       | LCHIJA | itemscroller                 | 0.12.0                             | itemscroller-1.12.2-0.12.0.jar                             | 2b03e1423915a189b8094816baa18f239d576dff |
       | LCHIJA | jeibees                      | 0.9.0.5                            | jeibees-0.9.0.5-mc1.12.2.jar                               | None                                     |
       | LCHIJA | journeymap                   | 1.12.2-5.7.0                       | journeymap-1.12.2-5.7.0.jar                                | None                                     |
       | LCHIJA | jaff                         | 1.7_for_1.12                       | JustAFewFish-1.7_for_1.12.jar                              | None                                     |
       | LCHIJA | loadmyresources              | 1.0.0                              | loadmyresources_1.0.0_MC_1.12-1.12.2.jar                   | None                                     |
       | LCHIJA | loliasm                      | 5.6                                | loliasm-5.6.jar                                            | None                                     |
       | LCHIJA | loottweaker                  | 0.3.0                              | LootTweaker-0.3.0+MC1.12.2.jar                             | None                                     |
       | LCHIJA | mobdismemberment             | 7.0.0                              | MobDismemberment-1.12.2-7.0.0.jar                          | None                                     |
       | LCHIJA | mrtjpcore                    | 2.1.4.43                           | MrTJPCore-1.12.2-2.1.4.43-universal.jar                    | None                                     |
       | LCHIJA | namepain                     | 1.4.0                              | namepain-1.4.0+forge-1.12.x.jar                            | None                                     |
       | LCHIJA | netherportalfix              | 5.3.13                             | NetherPortalFix_1.12.1-5.3.13.jar                          | None                                     |
       | LCHIJA | neid                         | 1.5.4.4                            | NotEnoughIDs-1.5.4.4.jar                                   | None                                     |
       | LCHIJA | nuclearcraft                 | 2o.6.2                             | NuclearCraft-2o.6.2-1.12.2.jar                             | None                                     |
       | LCHIJA | particleculling              | v1.3                               | particleculling-1.12.2-v1.3.1.jar                          | None                                     |
       | LCHIJA | patchouli                    | 1.0-20                             | Patchouli-1.0-20.jar                                       | None                                     |
       | LCHIJA | peacefulsurface              | 1.12.2-v1b                         | PeacefulSurface-1.12.2-v1b.jar                             | aaaf83332a11df02406e9f266b1b65c1306f0f76 |
       | LCHIJA | pneumaticcraft               | 1.12.2-0.11.15-398                 | pneumaticcraft-repressurized-1.12.2-0.11.15-398.jar        | None                                     |
       | LCHIJA | projectred-core              | 4.9.4.120                          | ProjectRed-1.12.2-4.9.4.120-Base.jar                       | None                                     |
       | LCHIJA | projectred-integration       | 4.9.4.120                          | ProjectRed-1.12.2-4.9.4.120-integration.jar                | None                                     |
       | LCHIJA | projectred-transmission      | 4.9.4.120                          | ProjectRed-1.12.2-4.9.4.120-integration.jar                | None                                     |
       | LCHIJA | projectred-illumination      | 4.9.4.120                          | ProjectRed-1.12.2-4.9.4.120-lighting.jar                   | None                                     |
       | LCHIJA | pdp                          | 1.12.2-1.2.4                       | ProportionalDestructionParticles-1.12.2-1.2.4.jar          | None                                     |
       | LCHIJA | qmd                          | 1.1.2                              | QMD-1.1.2-1.12.2.jar                                       | None                                     |
       | LCHIJA | quarkoddities                | 1                                  | QuarkOddities-1.12.2.jar                                   | None                                     |
       | LCHIJA | redstoneflux                 | 2.1.0                              | RedstoneFlux-1.12-2.1.0.6-universal.jar                    | 8a6abf2cb9e141b866580d369ba6548732eff25f |
       | LCHIJA | rsgauges                     | 1.2.4                              | rsgauges-1.12.2-1.2.4.jar                                  | ed58ed655893ced6280650866985abcae2bf7559 |
       | LCHIJA | rustic                       | 1.1.7                              | rustic-1.1.7.jar                                           | None                                     |
       | LCHIJA | rusticthaumaturgy            | 4.4a                               | Rustic Thaumaturgy-4.4a.jar                                | None                                     |
       | LCHIJA | savemystronghold             | 1.12.2-1.0.0                       | savemystronghold-1.12.2-1.0.0.jar                          | None                                     |
       | LCHIJA | sereneseasons                | 1.2.18                             | SereneSeasons-1.12.2-1.2.18-universal.jar                  | None                                     |
       | LCHIJA | serenetweaks                 | 0.0.1                              | SereneTweaks-1.0.jar                                       | None                                     |
       | LCHIJA | sgcraft                      | 2.0.3                              | SGCraft-2.1.0.jar                                          | None                                     |
       | LCHIJA | soundfilters                 | 0.12.1_for_1.12                    | SoundFilters-0.12.1_for_1.12.jar                           | None                                     |
       | LCHIJA | storagedrawers               | 1.12.2-5.4.0                       | StorageDrawers-1.12.2-5.4.0.jar                            | None                                     |
       | LCHIJA | sync                         | 7.0.2                              | Sync-1.12.2-7.0.2.jar                                      | None                                     |
       | LCHIJA | tcresearchpatcher            | 1.12.2-1.1.2                       | ThaumcraftResearchPatcher-1.12.2-1.1.2.jar                 | 8f678591ba6f78d579e553a8aa94b4c4766cb13d |
       | LCHIJA | thaumicaugmentation          | @VERSION@                          | ThaumicAugmentation-1.12.2-2.1.6.jar                       | None                                     |
       | LCHIJA | thaumicjei                   | 1.6.0                              | ThaumicJEI-1.12.2-1.6.0-27.jar                             | None                                     |
       | LCHIJA | thaumicspeedup               | 4.0                                | thaumicspeedup-4.0.jar                                     | None                                     |
       | LCHIJA | tips                         | 1.0.9                              | Tips-1.12.2-1.0.9.jar                                      | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJA | travelersbackpack            | 1.0.35                             | TravelersBackpack-1.12.2-1.0.35.jar                        | None                                     |
       | LCHIJA | tbintegration                | 1.0.1                              | TravelersBackpackIntegration-1.12.2-1.0.1.jar              | None                                     |
       | LCHIJA | traverse                     | 1.6.0                              | Traverse-1.12.2-1.6.0-69.jar                               | None                                     |
       | LCHIJA | worldstripper                | 1.6.0-1.12.2                       | World-Stripper-1.6.0-1.12.2.jar                            | None                                     |
       | LCHIJA | solcarrot                    | 1.8.4                              | solcarrot-1.12.2-1.8.4.jar                                 | None                                     |
       | LCHIJA | commons0815                  | 1.4.0                              | Commons0815-1.12.2-1.4.0.jar                               | None                                     |
       | LCHIJA | guitoolkit                   | 1.4.0                              | Commons0815-1.12.2-1.4.0.jar                               | None                                     |
       | LCHIJA | rendertoolkit                | 1.4.0                              | Commons0815-1.12.2-1.4.0.jar                               | None                                     |
       | LCHIJA | openglasses                  | @VERSION@-53                       | OpenGlasses-MC1.12.2-2.2-53.jar                            | None                                     |
       | LCHIJA | eleccoreloader               | 1.9.452                            | ElecCore-1.12.2-1.9.452.jar                                | None                                     |
  Loaded coremods (and transformers): GregTechLoadingPlugin (gregtech-1.12.2-2.4.1-beta.jar)
                                        gregtech.core.GregTechTransformer
                                      CreativePatchingLoader (CreativeCore_v1.10.61_mc1.12.2.jar)
                                        
                                      CorePlugin (SmoothFont-mc1.12.2-2.1.2.jar)
                                        bre.smoothfont.asm.Transformer
                                      ParticleCullingLoadingPlugin (particleculling-1.12.2-v1.3.1.jar)
                                        
                                      TransformerLoader (OpenComputers-MC1.12.2-1.7.6+fe5ba4e.jar)
                                        li.cil.oc.common.asm.ClassTransformer
                                      MicdoodlePlugin (MicdoodleCore-1.12.2-4.0.2.280.jar)
                                        micdoodle8.mods.miccore.MicdoodleTransformer
                                      Thaumcraft Research Patcher Core Plugin (ThaumcraftResearchPatcher-1.12.2-1.1.2.jar)
                                        thecodex6824.tcresearchpatcher.Transformer
                                      Quark Plugin (Quark-r1.6-179.jar)
                                        vazkii.quark.base.asm.ClassTransformer
                                      AppleCore (AppleCore-mc1.12.2-3.4.0.jar)
                                        squeek.applecore.asm.TransformerModuleHandler
                                      LoadingPlugin (Bloodmoon-MC1.12.2-1.5.3.jar)
                                        lumien.bloodmoon.asm.ClassTransformer
                                      EnderCorePlugin (EnderCore-1.12.2-0.5.76-core.jar)
                                        com.enderio.core.common.transform.EnderCoreTransformer
                                        com.enderio.core.common.transform.SimpleMixinPatcher
                                      BlurPlugin (Blur-1.0.4-14.jar)
                                        com.tterrag.blur.BlurTransformer
                                      ItemPatchingLoader (ItemPhysic_Full_1.4.34_mc1.12.2.jar)
                                        com.creativemd.itemphysic.ItemTransformer
                                      MixinBooter (!mixinbooter-5.0.jar)
                                        
                                      LoadingPlugin (HardcoreDarkness-MC1.12.2-2.0.jar)
                                        lumien.hardcoredarkness.asm.ClassTransformer
                                      LoliASM (loliasm-5.6.jar)
                                        
                                      Fluidlogged API Plugin (Fluidlogged-API-v1.8.1e-mc1.12.2.jar)
                                        git.jbredwards.fluidlogged_api.mod.asm.ASMHandler$Transformer
                                      LoadingPlugin (ChunkAnimator-MC1.12-1.2.jar)
                                        lumien.chunkanimator.asm.ClassTransformer
                                      RandomPatches (randompatches-1.12.2-1.22.1.10.jar)
                                        com.therandomlabs.randompatches.core.RPTransformer
                                      SSLoadingPlugin (SereneSeasons-1.12.2-1.2.18-universal.jar)
                                        sereneseasons.asm.transformer.EntityRendererTransformer
                                        sereneseasons.asm.transformer.WorldTransformer
                                      Persistency (persistency-1.2.0.jar)
                                        zone.rong.persistency.Persistency$ClientTransformer
                                      WitcheryLoadingPlugin (WitcheryResurrected-1.12.2-0.5.2.4-forge-all.jar)
                                        net.msrandom.witchery.asm.WitcheryLoadingPlugin$Transformer
                                      CTMCorePlugin (CTM-MC1.12.2-1.0.2.31.jar)
                                        team.chisel.ctm.client.asm.CTMTransformer
                                      FluidCraftCore (Fluid+Craft+for+AE2-2.3.3-r.jar)
                                        com.glodblock.github.coremod.FCClassTransformer
                                      CharmLoadingPlugin (Charm-1.12.2-1.4.1.jar)
                                        svenhjol.charm.base.CharmClassTransformer
                                      Plugin (NotEnoughIDs-1.5.4.4.jar)
                                        ru.fewizz.neid.asm.Transformer
                                      GSHooksLoader (GalaxySpace-1.12.2-2.0.18.jar)
                                        galaxyspace.core.hooklib.minecraft.PrimaryClassTransformer
                                      EntityCullingPlugin (EntityCulling-1.12.2-5.2.2.jar)
                                        meldexun.entityculling.asm.EntityCullingClassTransformer
                                      Thaumic Augmentation Core Plugin (ThaumicAugmentation-1.12.2-2.1.6.jar)
                                        thecodex6824.thaumicaugmentation.core.TATransformer
                                      Aqua Acrobatics Transformer (AquaAcrobatics-1.12.1.jar)
  GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
  AE2 Version: stable rv6-stable-7-extended_life-v0.52.7 for Forge 14.23.5.2847
  Ender IO: No known problems detected.
            Authlib is : /home/simon/.local/share/multimc/libraries/com/mojang/authlib/1.5.25/authlib-1.5.25.jar
            
            !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
            !!!You are looking at the diagnostics information, not at the crash.       !!!
            !!!Scroll up until you see the line with '---- Minecraft Crash Report ----'!!!
            !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
  forestry : Modules have been disabled in the config: Mystical Agriculture, Energy, Roots, Worktable, TechReborn, Better With Mods, Crate, Charcoal, Factory, ImmersiveEngineering, Mail, Actually Additions
  AE2 Integration: IC2:OFF, GTCE:ON, RC:OFF, MFR:OFF, Waila:OFF, InvTweaks:OFF, JEI:ON, Mekanism:OFF, OpenComputers:ON, THE_ONE_PROBE:OFF, TESLA:OFF, CRAFTTWEAKER:ON
  Suspected Mods: Fluidlogged API (fluidlogged_api)
  Profiler Position: N/A (disabled)
  Player Count: 1 / 8; [GCEntityPlayerMP['simon_kungen'/9111, l='New World', x=-147.73, y=66.13, z=281.69]]
  Type: Integrated Server (map_client.txt)
  Is Modded: Definitely; Client brand changed to 'fml,forge'

```
</details>

[latest.log](https://github.com/CleanroomMC/Fluidlogged-API/files/9471961/latest.log)

**Comment by Shazuli** [comment]
Looks like it happens sometimes when you place a block in the world too other than just generating a new world

**Comment by jbredwards** [comment]
looks like it's a Galaxy Space conflict. When that mod applies its transformer to `World.setBlockState`, it feels the need to shift the method's local variable indexes for some reason :thinking:. I'll add a workaround in the next version

**Comment by jbredwards** [comment]
fixed in v1.9.0.0
